testautoRIFT_ISCE.py

```diff
@@ -508,6 +510,12 @@ def generateAutoriftProduct(indir_m, indir_s, grid_location, init_offset, search
         m_name = os.path.basename(indir_m)
         s_name = os.path.basename(indir_s)
 
+        # FIXME: Filter width is a magic variable here and not exposed well.
```

Comment:
Maybe for now we just print the filter type and window size to screen

Reply:
@alex-s-gardner yep the filter width is printed here: https://github.com/nasa-jpl/autoRIFT/pull/69/files#diff-0da0f6848bb25f16593e48fa47fea8248d3dd9214f85b82e6a007381d8a1ffdaR289

I can also print the filter used as well.

This comment is for me to come back to (hopefully in Jan.) to actually expose this as a config value or cli/fn argument so that it can be adjusted for any run.